### PR TITLE
Refactor Code and Class Name

### LINKED ISSUE #69: [Refactor] Modify v1.2 code
## Details
- [x] Remove outdated class from Addressbook
- [x] Refactor class name and code to remove postfix "Trackermon"

### PR DESCRIPTION
**Pull request by ghost**
Remove outdated class from Addressbook
 Refactor class name and code to remove postfix "Trackermon"
Comment out currently unused class but useful in the future

### CODE REVIEW

File: src/main/java/seedu/address/model/Model.java [hunk 1]
```diff
@@ -5,14 +5,11 @@
 
 import javafx.collections.ObservableList;
 import seedu.address.commons.core.GuiSettings;
-import seedu.address.model.person.Person;
+import seedu.address.model.show.Show;
 
-/**
- * The API of the Model component.
- */
```
Comment (arcornior) [inline src/main/java/seedu/address/model/Model.java:12]: Should we keep this comment?

File: src/main/java/seedu/address/MainApp.java [hunk 2]
```diff
@@ -69,25 +69,25 @@ public void init() throws Exception {
     }
 
     /**
-     * Returns a {@code ModelManager} with the data from {@code storage}'s address book and {@code userPrefs}. <br>
-     * The data from the sample address book will be used instead if {@code storage}'s address book is not found,
-     * or an empty address book will be used instead if errors occur when reading {@code storage}'s address book.
+     * Returns a {@code ModelManager} with the data from {@code storage}'s show list and {@code userPrefs}. <br>
+     * The data from the sample show liost will be used instead if {@code storage}'s show list is not found,
```
Comment (JonathanHoshi) [inline src/main/java/seedu/address/MainApp.java:73]: There is a minor typo here. `show liost -> show list`

File: src/main/java/seedu/address/model/ModelManager.java [hunk 3]
```diff
@@ -11,33 +11,31 @@
 import javafx.collections.transformation.FilteredList;
 import seedu.address.commons.core.GuiSettings;
 import seedu.address.commons.core.LogsCenter;
-import seedu.address.model.person.Person;
+import seedu.address.model.show.Show;
 
-/**
- * Represents the in-memory model of the address book data.
- */
```
Comment (arcornior) [inline src/main/java/seedu/address/model/ModelManager.java:18]: Should we keep this comment too?

File: src/main/java/seedu/address/model/ModelManager.java [hunk 4]
```diff
@@ -63,69 +61,69 @@ public void setGuiSettings(GuiSettings guiSettings) {
         requireNonNull(guiSettings);
         userPrefs.setGuiSettings(guiSettings);
     }
-
+    //Done
```
Comment (arcornior) [inline src/main/java/seedu/address/model/ModelManager.java:64]: What's this comment for? Should it be deleted?

File: src/main/java/seedu/address/model/ReadOnlyUserPrefs.java [hunk 5]
```diff
@@ -4,13 +4,10 @@
 
 import seedu.address.commons.core.GuiSettings;
 
-/**
- * Unmodifiable view of user prefs.
- */
```
Comment (arcornior) [inline src/main/java/seedu/address/model/ReadOnlyUserPrefs.java:9]: Should we keep this comment too?

File: src/main/java/seedu/address/model/UserPrefs.java [hunk 6]
```diff
@@ -8,13 +8,10 @@
 
 import seedu.address.commons.core.GuiSettings;
 
-/**
- * Represents User's preferences.
- */
```
Comment (arcornior) [inline src/main/java/seedu/address/model/UserPrefs.java:13]: Should we keep this comment too?

File: src/main/java/seedu/address/logic/commands/ClearCommand.java [hunk 7]
```diff
@@ -1,23 +1,23 @@
...
+//    public static final String MESSAGE_SUCCESS = "Address book has been cleared!";
+//
+//
+//    @Override
+//    public CommandResult execute(Model model) {
+//        requireNonNull(model);
+//        model.setAddressBook(new AddressBook());
+//        return new CommandResult(MESSAGE_SUCCESS);
+//    }
+//}
```
Comment (JonathanHoshi) [inline src/main/java/seedu/address/logic/commands/ClearCommand.java:23]: Is there a need to comment out unused classes?

**Review by Ardentsoul — APPROVED** [review]
LGTM with some nitpicks.

**Review by arcornior — COMMENTED** [review]
LGTM, maybe keep the header comments for the component classes?
Do check your code for extra comments that should have been removed too!

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 8]
```diff
@@ -51,7 +52,7 @@
     private StackPane statusbarPlaceholder;
 
     /**
-     * Creates a {@code MainWindow} with the given {@code Stage} and {@code Logic}.
+     * Creates a {@code MainWindowTrackermon} with the given {@code Stage} and {@code LogicTrackermon}.
```
Comment (arcornior) [inline src/main/java/seedu/address/ui/MainWindow.java:55]: Should this be changed to 
``* Creates a {@code MainWindow} with the given {@code Stage} and {@code Logic}.`` instead? We should have removed all the Trackermon suffix classes already.

**Review by JonathanHoshi — COMMENTED** [review]
Overall good job in refactoring the classes!

File: src/main/java/seedu/address/logic/commands/Command.java [hunk 9]
```diff
@@ -11,7 +11,7 @@
     /**
      * Executes the command and returns the result message.
      *
-     * @param model {@code Model} which the command should operate on.
+     * @param model {@code ModelTrackermon} which the command should operate on.
```
[the remark this answers is not in the document]
Comment (lowkaiwei98) [reply src/main/java/seedu/address/logic/commands/Command.java]: Looks i missed a bunch of a comment. Sorry did it in a rush so yall can start asap.

File: src/main/java/seedu/address/logic/commands/CommandResult.java [hunk 10]
```diff
@@ -18,7 +18,7 @@
     private final boolean exit;
 
     /**
-     * Constructs a {@code CommandResult} with the specified fields.
+     * Constructs a {@code CommandResultTrackermon} with the specified fields.
```
[the remark this answers is not in the document]
Comment (lowkaiwei98) [reply src/main/java/seedu/address/logic/commands/CommandResult.java]: nice catch

File: src/main/java/seedu/address/model/Model.java (hunk 1 above)
Comment (lowkaiwei98) [reply src/main/java/seedu/address/model/Model.java]: ya i might have delete by accident

File: src/main/java/seedu/address/model/ModelManager.java (hunk 3 above)
Comment (lowkaiwei98) [reply src/main/java/seedu/address/model/ModelManager.java]: yup

File: src/main/java/seedu/address/model/ModelManager.java (hunk 4 above)
Comment (lowkaiwei98) [reply src/main/java/seedu/address/model/ModelManager.java]: Not sure how it end up there

File: src/main/java/seedu/address/model/ReadOnlyUserPrefs.java (hunk 5 above)
Comment (lowkaiwei98) [reply src/main/java/seedu/address/model/ReadOnlyUserPrefs.java]: yup

File: src/main/java/seedu/address/model/UserPrefs.java (hunk 6 above)
Comment (lowkaiwei98) [reply src/main/java/seedu/address/model/UserPrefs.java]: yup

File: src/main/java/seedu/address/MainApp.java (hunk 2 above)
Comment (lowkaiwei98) [reply src/main/java/seedu/address/MainApp.java]: yup

File: src/main/java/seedu/address/logic/commands/ClearCommand.java (hunk 7 above)
Comment (lowkaiwei98) [reply src/main/java/seedu/address/logic/commands/ClearCommand.java:23]: Unless you want to me to refactor all the dependencies? I still think keeping a old copy for ref is better.

File: src/main/java/seedu/address/ui/MainWindow.java (hunk 8 above)
Comment (lowkaiwei98) [reply src/main/java/seedu/address/ui/MainWindow.java]: Of course i know we should remove all the Trackmon suffix... obviously i just missed this one.

**Review by arcornior — APPROVED** [review]
LGTM!

File: src/main/java/seedu/address/logic/commands/ClearCommand.java (hunk 7 above)
Comment (JonathanHoshi) [reply src/main/java/seedu/address/logic/commands/ClearCommand.java:23]: I see, no worries just wanted to know the reason. Good job!

**Pull request merged**